integration-tests/mysql-client-tests/import-mysqldump.bats

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env bats
+
+setup() {
+    REPO_NAME="dolt_repo_$$"
+    mkdir $REPO_NAME
+    cd $REPO_NAME
+
+    dolt init
+}
+
+teardown() {
+    cd ..
+    rm -rf $REPO_NAME
+}
+
+@test "import mysqldump: empty database dump" {
+    service mysql start
```

Comment:
I think you can move start and stop to setup and teardown

Reply:
I kept this inside the test since we use mysql-server in only two tests, others I test on parts of mysqldump code without using mysql server or mysqldump